system/libraries/Session/drivers/Session_redis_driver.php

```
@@ -272,7 +274,8 @@ public function close()
 		if (isset($this->_redis))
 		{
 			try {
-				if ($this->_redis->ping() === '+PONG')
+				$ping = $this->_redis->ping();
+				if ($ping === '+PONG' OR $ping === TRUE)
```

Why would we silently accept both if we know the version?

---

Solved in the latest commit.